jail-events.yml

```diff
@@ -5,27 +5,9 @@
   connection: local
   become: yes
   vars:
-    env_name: venv
     web_host_name: "events.buildbot.net"
 
   roles:
   - base
-  - role: user
```

Comment:
The user, virtualenv, and supervisor-service roles don't have a replacement now. I think the user block should remain. Is the virtualenv and supervisor-service parts no longer needed?